tracking/third-party/chartbeat.js

```diff
@@ -5,7 +5,15 @@ In this case we are only interested in the editorial; specifically the HUD (head
 Eventually this will be replaced by FT's in-house "lantern" application, utilising our own data.
 */
 
-import {loadScript, getCookieValue} from 'n-ui-foundations';
+import {cookieStore} from 'n-ui-foundations';
+
+const loadScript = (src) => {
```

Comment:
this exists in https://github.com/Financial-Times/n-ui/pull/814/files#diff-8ee9c98ed98c3da44bd3d077cb925d38 now
(sorry for lack of documentation... it'll come)